apilayer/handler/InventoriesHandler.go

```diff
@@ -242,6 +242,50 @@ func UpdateSelectedInventory(rw http.ResponseWriter, r *http.Request, user strin
 	json.NewEncoder(rw).Encode(resp)
 }
 
+// GetAllInventoriesAssociatedWithSelectEvent ...
+// swagger:route GET /api/v1/profile/{eventID}/associated-inventories GetAllInventoriesAssociatedWithSelectEvent getAllInventoriesAssociatedWithSelectEvent
+//
+// # Retrieves the list of inventories for each users
+//
+// # Updates the selected notification
+//
+// Parameters:
+//   - +name: eventID
+//     in: path
+//     description: The eventID of the selected event
+//     type: string
+//     required: true
+//
+// Responses:
+// 200: []Inventory
+// 400: MessageResponse
+// 404: MessageResponse
+// 500: MessageResponse
+func GetAllInventoriesAssociatedWithSelectEvent(rw http.ResponseWriter, r *http.Request, user string) {
+
+	vars := mux.Vars(r)
+
+	eventID := vars["eventID"]
+
+	if len(eventID) <= 0 {
```

Comment:
eventID should be of a UUID variant. if the uuid variant is not found we should throw an error. missing step.